### ISSUE TITLE
Failure signing RPMs using chainbuild 

### ISSUE THREAD

**Issue by mnencia** [alanfranz/docker-rpm-builder#40]
The build works perfectly using  dir command.

The error is
```
+ echo -e '\n'
+ setsid rpmsign --addsign
rpmsign: no arguments given
```

**Comment by mnencia** [comment]
That's because the `rpmbuild -bs` output is different and the sed expression don't find the .src.rpm name.

```
rpmbuild_out='Wrote: /root/rpmbuild/SRPMS/testpkg-1.0-1.src.rpm'
```

**Comment by alanfranz** [comment]
Fix and test included in 1.38. Next time, if you could include an automated test, it would speed up merging.

Thanks!
